app/frontend/Calendar/EditEvent/ReminderBox.svelte

```diff
@@ -27,16 +27,9 @@
 
   let durationUnit: DurationUnit;
   let durationInUnit: number;
-  $: beforeInSec = $event.alarm ? ($event.startTime.getTime() - $event.alarm.getTime()) / 1000 : 0;
+  let beforeInSec = $event.alarm ? ($event.startTime.getTime() - $event.alarm.getTime()) / 1000 : 0;
 
-  function onChanged(ev: CustomEvent<number>) {
-    let seconds = ev.detail;
-    if (!event.alarm) {
-      event.alarm = new Date();
-    }
-    event.alarm.setTime(event.startTime.getTime() - seconds * 1000);
-    event.notifyObservers();
-  }
```

Comment:
hm, this is some very odd code, so this was definitely deliberately there, to fix a certain case/bug.
Are you sure you tested all cases, and that this is not needed? If so, all the better. I Just want to make sure that this is not regressing a bugfix.